.goreleaser.yaml

```diff
@@ -0,0 +1,59 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+
+# The lines bellow are called `modelines`. See `:help modeline`
+# Feel free to remove those if you don't want/need to use them.
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    # - go mod tidy
+    # you may remove this if you don't need go generate
+    # - go generate ./...
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      # - windows
```

Comment:
Currently there is a problem with windows build:
````
../../../go/pkg/mod/github.com/pkg/term@v1.1.0/term_windows.go:73:8: undefined: attr
../../../go/pkg/mod/github.com/pkg/term@v1.1.0/term_windows.go:74:12: undefined: termios
../../../go/pkg/mod/github.com/pkg/term@v1.1.0/term_windows.go:74:40: t.fd undefined (type *Term has no field or method fd)
../../../go/pkg/mod/github.com/pkg/term@v1.1.0/term_windows.go:74:47: undefined: syscall
````

so it might require an additional tweak later on.